app/views/vendors/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1> Add a Vendor </h1>
+<%= simple_form_for @vendor do |f| %>
+<%= f.error_notification %>
+
+  <%= hidden_field_tag :next, params[:next] %>
```

Comment:
What does this line do ?

Reply:
When the new vendor form is submitted, this gives it the ability to redirect to the new items page.